spec/features/system_admin/uploads/delete_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Delete a file" do
+  let!(:user) { create(:user, :system_admin) }
+  let!(:upload) { create(:upload) }
+
+  let(:index) { admin_uploads_page }
```

Comment:
There's no  need to wrap the methods in `let` blocks as the page object is memoised. Just the method directly.

Reply:
I prefer this aliasing